submissions/cammer87.md

```diff
@@ -0,0 +1,17 @@
+* submitter: @cammer87
+* statement: I hereby agree that the facts I have submitted are my own, and that I grant a license for Pryaxis Limited, or designated entities by Pryaxis Limited, the ability to use this data in a way to promote future Pryaxis services.
+
+----
+
+* @IsleRoyaleNationalPark is the least visited national park in the United States.
+* The wolf population of @IsleRoyaleNationalPark is dying off due to genetic inbreeding.
+* @Isle Royale National Park is only accessible by boat or seaplane.
```

Comment:
Heya, everything is great except this needs to either be ```@IsleRoyaleNationalPark``` or the rest need to be ```@Isle Royale National Park```.